src/components/textfield/textfield.scss

```diff
@@ -0,0 +1,92 @@
+@import '~theme/base/variables';
```

Comment:
I notice you reuse a lot of the same styles. This might be a good opportunity to use `mixins`, `functions` or combine some `selectors` as well as save some numbers to variables, like width for example.

Reply:
Until now, we are using it only for colours if I am not wrong.
We *should create a utils too* for that *standard way* to generate the spacing (padd/marg) through `mixins` and `composition`. We should create a `composition` too for the generalAligment (and refactor the name I gave it if any one knows a better name for it)